akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -476,15 +480,32 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
           }
         }
 
+      def assertExpectedMdc(event: LoggingEvent) = {
```

Comment:
how is this different from `LoggingTestKit.withMdc`?

Reply:
We can't use that because the actor path isn't known until it is started, so that is done with an atomic in this test (this doesn't change that)